### DISCUSSION TITLE
Marp Theme - Themes created with simplicity and beauty in mind.

### DISCUSSION THREAD

**Discussion by cunhapaulo** [marp-team/marp#232] [Show and tell]
Hi folks,

Must say my admiration for Marp and its Team grows every day I move on using it! Congratulations and thanks for this marvelous work!

Just created some themes for Marp and decided they could be of use to someone else besides me. So it is now shared to everyone who wishes to give it a try. It is still an evolving project, but at least three themes are redy for use (Socrates, Plato and Hegel). Soon Kant will follow (although Hegel followed Kant in German Idealism).

[https://github.com/cunhapaulo/marpstyle](https://github.com/cunhapaulo/marpstyle)

Here is a small screenshot of this Plato theme:

![plato theme](https://github.com/cunhapaulo/marpstyle/blob/main/img/plato.png?raw=true)

Hope this work comes to be useful!

Bye.

**Comment by cheind** [comment]
Now, the Latex-Beamer replacement is complete :) Thanks for the styles, they look very clean - in particular the two-column classes look helpful.

**Reply by cunhapaulo** [reply]
There is much more to come.

**Reply by cheind** [reply]
Well, I'm already happy with what is currently possible. Sometimes its better to accept the lack of features and instead rethink the way you want to present.

**Reply by cunhapaulo** [reply]
Simplicity and beauty walk together, hand in hand!. Thanks for your comments.

**Reply by Juanvvc** [reply]
Thanks for the styles! They seem very clean and elegant, I will definitely use some or your ideas

**Reply by cunhapaulo** [reply]
I am grateful for your contribuitions too! Gracias and thanks for your comments!

**Comment by cunhapaulo** [comment]
I decided to separate CSS structure from the color schema in order to make customization easier. Hope this will help the creation of many new themes!!!

**Comment by cunhapaulo** [comment]
There are some new themes available worth visiting.

https://github.com/cunhapaulo/marpstyle

**Reply by puat133** [reply]
Hi, thanks for these themes, they look awesome.
One quick question: how to enable math expression (katex) inside figure captions?

**Reply by cunhapaulo** [reply]
Hello puat133,

I am happy these styles may be useful.

The way I managed to do caption for formulae was using  `<figcaption>`

Like this:

```markdown
# LaTeX Equations

$$\frac{1}{c^2}\frac{\partial^2\mathbf{\psi}}{\partial t^2} = \nabla^2 \circ\mathbf{\psi} $$
$$\nabla \times \textbf{E}=- \frac{\partial \textbf{B}}{\partial t}$$
$$\nabla^2 \textbf{E} = \mu \epsilon \frac{\partial^2 \textbf{E}}{\partial t^2}$$
$$c=\sqrt{\frac{1}{\mu \epsilon}}$$

<figcaption align="center">
<b>Formulae</b>: Exemples of use of LaTeX formulas.
</figcaption>
```

**Reply by puat133** [reply]
Hi, thanks for the reply. What I would like to do is something like this:
```
<figcaption align="center">
<b>Formulae</b>: Examples of use of LaTeX formulas: $e=mc^2$.
</figcaption>
```

In my present setup, the math expression in the caption will not rendered correctly. Is this kind of expression possible?

**Reply by yhatt** [reply]
Hey @puat133, have a look at https://github.com/marp-team/marp/discussions/152#discussioncomment-1218998. Math syntax `$` is a part of Markdown contents so you can render it with HTML by [CommonMark compliant way](https://spec.commonmark.org/0.30/#:~:text=it%20provides%20a%20very%20simple%20and%20flexible%20way%20of%20including%20Markdown%20content%20inside%20HTML%20tags%3A%20simply%20separate%20the%20Markdown%20from%20the%20HTML%20using%20blank%20lines%3A).

**Reply by puat133** [reply]
Great, it works. Thanks.

**Reply by cunhapaulo** [reply]
All you have to do is to provide at least one blank line after the first \<figcaption\> mark!!!
Just like this:

```html

<figcaption align="center">

<b>Formulae</b>: Examples of use of LaTeX formulas: $e=mc^2$.

</figcaption>

```

**Reply by cunhapaulo** [reply]
![image](https://user-images.githubusercontent.com/28146759/154156166-0b407a92-f99f-4d57-9e30-88bb89428ec7.png)

**Reply by puat133** [reply]
Thanks.
